backend/config.sample.yml

```
@@ -15,6 +15,10 @@ server_sessions:
   expires_in: 3600
 
 matrix_bot:
+  # Details of the user account which will be acting as the bot for the application service.
```

An application service is a specific matrix thing, which we are not.
```suggestion
  # Details of the user account which will be acting as the bot for the application.
```